File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/recommended_fee/RecommendedFeeParentPresenter.java
package io.muun.apollo.presentation.ui.fragments.recommended_fee;

import io.muun.apollo.domain.model.PaymentRequest;
import io.muun.apollo.presentation.ui.base.ParentPresenter;

public interface RecommendedFeeParentPresenter extends ParentPresenter {

    PaymentRequest getPaymentRequest();

    void confirmFee(double selectedFeeOption);

    void editFeeManually();
}

File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/recommended_fee/RecommendedFeeParentPresenter.kt
package io.muun.apollo.presentation.ui.fragments.recommended_fee

import io.muun.apollo.presentation.ui.base.ParentPresenter
import newop.EditFeeState
import rx.Observable

interface RecommendedFeeParentPresenter : ParentPresenter {

    fun watchEditFeeState(): Observable<EditFeeState>

    fun confirmFee(selectedFeeRateInVBytes: Double)

    fun editFeeManually()
}